docs/orchestration/server/telemetry.md

```diff
@@ -0,0 +1,18 @@
+# Telemetry
+
+Prefect Server sends usage telemetry and statistics to Prefect Technologies, Inc. All information collected in anonymous. We use this information to better understand how Prefect Server is used and to ensure that we're supporting active versions.
+
+To opt-out of telemetry, simply add the following to your user configuration file (see [user configuration](../../core/concepts/configuration.md#user-configuration)).
```

Comment:
```suggestion
To opt-out of telemetry, add the following to your user configuration file (see [user configuration](../../core/concepts/configuration.md#user-configuration)).
```

"Simply" can be off-putting in documentation (see https://jameshfisher.com/2017/02/22/dont-use-simply/)